src/components/MenuItem.tsx

```diff
@@ -72,7 +72,7 @@ type MenuItemProps = (ResponsiveProps | UnresponsiveProps) &
         badgeText?: string;
 
         /** Used to apply offline styles to child text components */
-        style?: ViewStyle;
+        style?: ViewStyle | ViewStyle[];
```

Comment:
Is it possible to use StyleProp<ViewStyle> here?

Reply:
Yes, definitely should be `StyleProp<ViewStyle>`